src/commands/delete/mod.rs

```diff
@@ -0,0 +1,201 @@
+pub mod api;
+
+use cloudflare::endpoints::account::Account;
+use cloudflare::endpoints::workers::{DeleteDurableObject, DeleteScript, WorkersScript};
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::response::ApiFailure;
+use cloudflare::framework::HttpApiClient;
+use prettytable::{Cell, Row, Table};
+
+use crate::commands::delete::api::{fetch_bindings, fetch_scripts};
+use crate::commands::whoami::fetch_accounts;
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::terminal::interactive;
+use crate::terminal::message::{Message, StdOut};
+
+use std::collections::{HashMap, HashSet};
+
+// Wrapper for `fn script_error`
+fn format_error(e: ApiFailure) -> String {
+    http::format_error(e, Some(&script_errors))
+}
+
+// Provides more detailed explanations of API error codes.
+fn script_errors(error_code: u16) -> &'static str {
+    match error_code {
+        10000 => "Your authentication might be expired or invalid. Please run `wrangler login` or `wrangler config` to authorize Wrangler",
+        10007 => "The script could not be found. Please make sure that that the script being deleted exists.",
+        10064 => "The Durable Objects namespaces need to be deleted before this script can be deleted.",
+        _ => "",
+    }
+}
+
+// Interactive mode
+pub fn run(user: &GlobalUser) -> Result<(), anyhow::Error> {
+    // Fetch and display user accounts
+    let accounts = fetch_accounts(user)?;
+    let (valid_accounts, table) = format_accounts(accounts)?;
+    println!("{}", &table);
+    let account_name = interactive::get_user_input(
+        "Please enter the name of the account you wish to delete a Workers script from",
+    );
+    if !valid_accounts.contains_key(&account_name) {
+        anyhow::bail!("Account name doesn't match.")
+    }
+
+    //  Fetch and display scripts related to the account
+    let account_id = valid_accounts.get(&account_name).unwrap();
```

Comment:
This seems really different from how IDs are handled everywhere else. This should use `manifest.account_id.load()` instead.